content/news/052/index.md

```diff
@@ -84,6 +85,21 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Other News
 
+### [Metalmancy @ OpenSauce][metalmancy_interview]
+
+![image/GIF description](metalmancy_interview.png)
+_Arcade cabinet close up_
+
+[Framework][framework_website] interviewed Carlos and Joanna on their games
+and arcade machines at [OpenSauce][opensauce_website].
```

Comment:
```suggestion
[Framework][framework_website] interviewed Carlos and Joanna on their games
and arcade machines at [OpenSauce][opensauce_website]
at their stand for [Metalmancy][metalmancy_website].
```

Reply:
Not sure exactly, it just feels a bit weird that metalmancy is not introduced

Reply:
Agreed. I wrote a little intro :)